rust/chains/tw_solana/src/modules/insert_instruction.rs

```diff
@@ -91,6 +91,30 @@ pub trait InsertInstruction {
         Ok(account_added_at)
     }
 
+    /// Adds a fee payer account to the message.
+    /// Note: The fee payer must NOT be in the account list yet.
+    fn add_fee_payer(&mut self, account: SolanaAddress) -> SigningResult<()> {
+        if self.account_keys_mut().contains(&account) {
+            // For security reasons, we don't allow adding a fee payer if it's already in the account list.
```

Comment:
Could you please clarify why don't we need to allow this?

Reply:
I have added some considerations in the comments of the code.